brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-2201 Typography-letter-spacing for number and heading of counter block

### Description
Added letter-spacing to Typography of number and heading of counter block

### Screenshots
https://d.pr/v/p5xyQa

### Types of changes
New feature that adds letter-spacing settings to both number heading of counter block

### How has this been tested?

- Go to styles in editor and in number typography will get to see letter spacing give value and see the changes.
- Repeat the same for heading typography.
- switch between tablet, mobile and desktop and give values changes will be reflected in frontend.

### Checklist:
